### ISSUE TITLE
Incorrect SPDX ID

### ISSUE THREAD

**Issue by kzantow** [anchore/syft#494]
**What happened**:
SPDX output should have a "unique" id, but instead the documentation is output:
> SPDXID: SPDXRef-DOCUMENT

should be something like:

> SPDXID: \<Some-unique-identifier>-DOCUMENT

**What you expected to happen**:
Correct SPDXID.

**How to reproduce it (as minimally and precisely as possible)**:
Any Syft command.

**Anything else we need to know?**:

**Environment**:
- Output of `syft version`:
- OS (e.g: `cat /etc/os-release` or similar):

**Comment by spiffcs** [comment]
I found the docs in the specification for this [here](https://spdx.github.io/spdx-spec/2-document-creation-information/#23-spdx-identifier).

I did a quick query on github and found their repo that has an example of 2.2 [here](https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXJSONExample-v2.2.spdx.json).

It looks like the format they use to build the reference looks like this
```
SPDXID : SPDXRef-fromDoap-1
SPDXID : SPDXRef-Saxon
```

The top level of the document [here](https://github.com/spdx/spdx-spec/blob/dcdb5bd76d913e1311715e43a0ea8125f95447b9/examples/SPDXJSONExample-v2.2.spdx.json#L2-L11)

```
The `SPDXID` is listed as `SPDXRef-DOCUMENT`.
The `name` is listed as `SPDX-Tools-v2.0`.
```

If someone were to build a reference to this doc would it be `SPDXRef-SPDX-Tools-v2.0`?

We currently use `internal/presenter/packages/test-fixtures/snapshot/TestSPDXJSONImagePresenter.golden` as our fixture.

```
 "SPDXID": "SPDXRef-DOCUMENT",
 "name": "user-image-input",
 "spdxVersion": "SPDX-2.2",
 "creationInfo": {
  "created": "2021-06-23T17:48:32.7379Z",
  "creators": [
   "Organization: Anchore, Inc",
   "Tool: syft-[not provided]"
  ],
  "licenseListVersion": "3.13"
 },
```

If someone were to build a `Ref` to this would it be `SPDXRef-user-image-input`?

Does part of the specification show we should be doing `<Some-unique-identifier>-DOCUMENT`?
